### DISCUSSION TITLE
Moes BHT-002 - what are the zigbee commands to select external sensor

### DISCUSSION THREAD

**Discussion by simondsmason** [Koenkk/zigbee2mqtt#17878] [Devices]
Hi, I am using these thermostat with direct zigbee connection - not using zigbee2mqtt.  I don't see it reporting the external sensor normally and I can't find documentation on how to switch it to the external sensor.  So I was hoping that the person that wrote this driver might be able to provide the documentation?  Thanks.

**Comment by kkossev** [comment]
https://github.com/Koenkk/zigbee-herdsman-converters/issues/1905
https://github.com/Koenkk/zigbee-herdsman-converters/pull/1912/commits/872f5e4299f33866ce3108f864720e7b9703b62e
```
....
moesSensor: 43,
....
```

**Reply by simondsmason** [reply]
Thanks @kkossev.  I see that the code references the GC model.  I have the GB electric floor heating model and as far as I can tell it does not appear to be reporting the external sensor temp through Zigbee.  I don't see any discussions about the GB model?  Thanks.

**Comment by soulcrash14** [comment]
Hey @simondsmason,
have you ever found a solution to change the sensor on a GC model? thanks!

**Reply by simondsmason** [reply]
No - no luck.  I gave up and tossed it.  I then got the AVATTO Tuya WiFi/ZigBee3.0 Smart Thermostat.  I was able to get this to work for my application - just reading the external temp sensor.  I use a Sinope thermostat to control the floor.  I had two sensors installed when they put the floor in.  One is connected to the Sinope and the second is connected to this thermostat.  Works great.

The item I purchased:
https://www.aliexpress.us/item/3256806253562437.html?spm=a2g0o.order_list.order_list_main.57.1a781802NGOliu&gatewayAdapt=glo2usa
